### ISSUE TITLE
Is this crate still maintained?

### ISSUE THREAD

**Issue by Ben-Lichtman** [darfink/detour-rs#38]
For those just seeing this - I would recommend moving to https://github.com/Hpmason/retour-rs - this seems better maintained.

**Comment by darfink** [comment]
Yes — [0.9.0](https://github.com/darfink/detour-rs/releases/tag/v0.9.0) is out: stable Rust, AArch64 & Apple silicon support, `no_std`, and updated dependencies. See the [changelog](https://github.com/darfink/detour-rs/blob/master/CHANGELOG.md).
